stable/kommander/templates/namespace-labeler.yaml

```diff
@@ -0,0 +1,62 @@
+{{- if .Values.namespaceLabels }}
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  namespace: {{ .Release.Namespace }}
+  name: namespace-label
+  annotations:
+    "helm.sh/hook": pre-install, pre-upgrade
+    "helm.sh/hook-weight": "-5"
+    "helm.sh/hook-delete-policy": hook-succeeded,before-hook-creation
+spec:
+  template:
+    spec:
+      serviceAccountName: namespace-label
+      containers:
+        - name: namespace-label
+          image: "bitnami/kubectl:1.19.7"
+          command: ["kubectl"]
+          args: ["label", "namespace", "{{ .Release.Namespace }}", {{ join "," .Values.namespaceLabels  }} ]
```

Comment:
This job will fail if the namespace does not exist. That _is_ what we want, though, isn't it?

Reply:
Yes it is, in addition if we do this post-install we would defeat the purpose of adding the label to the namespace. Do you suspect that the .Release.Namespace may be empty? such as in the default case?